docs/index.md

```diff
@@ -213,6 +362,7 @@ for _ in range(N_EPOCHS):
 with torch.no_grad():
     y_final = exp_fn(params, {'x': x})
 
+plt.figure()
```

Comment:
In my opinion, printing plots directly on the doc is more interesting than imposing copying and executing the code. I find it easier to understand that way. The work already done by @dominikandreasseitz prompted me to take this initiative.

In this example that was written by @dominikandreasseitz, I just added `plt.figure()` because the fact of not delimited the figures created errors to display them.